lib/sso_users_api/manager_job.rb

```diff
@@ -1,30 +1,16 @@
-require 'active_job'
+require 'sidekiq'
 
 module SsoUsersApi
-  class ManagerJob  < ActiveJob::Base
-    queue_as :sso_users_api
+  class ManagerJob
+    include Sidekiq::Worker
 
-    def self.delay_amount
-      2
-    end
+    sidekiq_options queue: :high_priority, retry: 0
 
-    def perform(id, class_name, count = 0, options = {})
```

Comment:
The retry functionality has been removed, since Sidekiq will handle this natively.